bytecomp/meta.ml

```diff
@@ -26,5 +26,3 @@ external release_bytecode : bytecode -> unit
                                  = "caml_static_release_bytecode"
 external invoke_traced_function : Obj.raw_data -> Obj.t -> Obj.t -> Obj.t
                                 = "caml_invoke_traced_function"
-external get_section_table : unit -> (string * Obj.t) list
-                           = "caml_get_section_table"
```

Comment:
This could still be done, unless there's a reason to keep it I'm missing? Regardless, the comment in L664 of `startup_byt.c` before calling `caml_init_section_table` should be updated.